tests/h/tweens_test.py

```diff
@@ -44,13 +44,15 @@ def test_it_redirects_for_redirected_routes(self, pyramid_request):
         ]
 
         pyramid_request.path = "/foo"
+        handler = create_autospec(lambda req: None)  # pragma: no cover
 
         tween = tweens.redirect_tween_factory(
-            lambda req: req.response, pyramid_request.registry, redirects
+            handler, pyramid_request.registry, redirects
         )
```

Comment:
The point of this test is it doesn't call the passed item. I've separated this out as a mock and asserted this now to make this clearer.